task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace task_DEV_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter A Number :");
+            int inputNumber = int.Parse(Console.ReadLine());
+
+            if (inputNumber % 2 == 0 && inputNumber % 5 == 0)
+            {
+                Console.WriteLine("Tutti-Frutti");
+            }
+            else if (inputNumber % 2 == 0)
+            {
+                Console.WriteLine("Tutti");
+            }
+            else if (inputNumber % 5 == 0)
+            {
+                Console.WriteLine("Frutti");
+            }
+            else if (!(inputNumber % 2 == 0) && !(inputNumber % 5 == 0))
```

Comment:
The both of this conditions are checked in code above that's why `else if` block is overkill. You can use just `else` block instead.

Reply:
Code style violation. Missed `{}`